### PR TITLE
docs: update README and added DockerHub badge

### PR DESCRIPTION
**Pull request by juzhiyuan**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Update README's latest release description;
- Added DockerHub badge

### CODE REVIEW

File: docs/deploy-with-docker.md [hunk 1]
```diff
@@ -19,7 +19,9 @@
 
 # Building and Launching with Docker
 
-To build a Dashboard with Docker, you simply download the `Dockerfile` file from the **root directory** to your device (no need to download all source codes) then follow this guide.
+**NOTE:** We support official Docker Image, please visit [DockerHub](https://hub.docker.com/r/apache/apisix-dashboard) for more information. The following steps are for building Docker Image manually.
+
+To build the Dashboard with Docker, you simply download the `Dockerfile` file from the **root directory** to your device (no need to download all source codes) then follow this guide.
```
Comment (imjoey) [inline docs/deploy-with-docker.md:24]: How about provide a download link for this `Dockerfile`?

Comment (juzhiyuan) [reply docs/deploy-with-docker.md:24]: 🤔 We may use this repo[1] to manage all products' Dockerfile in M2.4, let me create an issue to track this.

[1] https://github.com/apache/apisix-docker

Comment (juzhiyuan) [reply docs/deploy-with-docker.md:24]: https://github.com/apache/apisix-dashboard/issues/1265

**Review by nic-chen — APPROVED** [review]

**Review by moonming — COMMENTED** [review]
I think you can add "how to install from dockerhub" in readme

**Comment by juzhiyuan** [comment]
hmm I put both deploys with Docker and deploy with DockerHub in the `deploy-with-docker.md`[1].

[1] https://github.com/apache/apisix-dashboard/pull/1264/files#diff-307353982a1aa87643c2aa59342dc8b0c4e285f1d574028ee8ba100a32eeee0cR22

**Review by liuxiran — APPROVED** [review]

File: docs/deploy-with-docker.md [hunk 2]
```diff
@@ -19,7 +19,9 @@
 
 # Building and Launching with Docker
 
-To build a Dashboard with Docker, you simply download the `Dockerfile` file from the **root directory** to your device (no need to download all source codes) then follow this guide.
+**NOTE:** We support official Docker Image, please visit [DockerHub](https://hub.docker.com/r/apache/apisix-dashboard) for more information. The following steps are for building Docker Image manually.
```
Comment (gxthrj) [inline docs/deploy-with-docker.md:22]: docker image cannot claim to be an official image.

Ref to APISIX.

image

Comment (juzhiyuan) [reply docs/deploy-with-docker.md:22]: fine

**Pull request merged**